src/foundry/client/pixi/core/containers/sprite-mesh.d.mts

```diff
@@ -28,35 +28,181 @@ declare global {
+     * @defaultValue `0`
+     */
+    protected _width: number;
 
-    /** @defaultValue `-1` */
+    /**
+     * The height of the sprite (this is initially set by the texture).
+     * @defaultValue `0`
+     */
+    protected _height: number;
+
+    /**
+     * The texture that the sprite is using.
+     */
+    protected _texture: PIXI.Texture;
+
+    /**
+     * The texture ID
+     * @defaultValue `-1`
+     */
+    protected _textureID: number;
+
+    /**
+     * Cached tint value so we can tell when the tint is changed.
+     */
+    protected _cachedTint: [red: number, green: number, blue: number, alpha: number]
+
+    /**
+     * The texture trimmed ID.
+     * @defaultValue `-1`
+     */
     _textureTrimmedID: number;
 
-    /** @defaultValue `-1` */
-    _transformTrimmedID: number;
+    /**
+     * This is used to store the uvs data of the sprite, assigned at the same time
+     * as the vertexData in calculateVertices().
+     */
+    protected uvs: Float32Array;
 
     /**
-     * Forced to false for SpriteMesh
-     * @defaultValue `false`
+     * The anchor point defines the normalized coordinates
+     * in the texture that map to the position of this
+     * sprite.
+     *
+     * By default, this is `(0,0)` (or `texture.defaultAnchor`
+     * if you have modified that), which means the position
+     * `(x,y)` of this `Sprite` will be the top-left corner.
+     *
+     * Note: Updating `texture.defaultAnchor` after
+     * constructing a `Sprite` does _not_ update its anchor.
+     *
+     * {@link https://docs.cocos2d-x.org/cocos2d-x/en/sprites/manipulation.html}
      */
-    override _roundPixels: boolean;
+    protected _anchor: PIXI.ObservablePoint;
 
-    /** @defaultValue `null` */
-    vertexTrimmedData: null;
+    /**
+     * This is used to store the vertex data of the sprite (basically a quad).
+     */
+    protected vertexData: Float32Array;
 
-    override isSprite: boolean;
+    /**
+     * This is used to calculate the bounds of the object IF it is a trimmed sprite.
+     * @defaultValue `null`
+     */
+    protected vertexTrimmedData: Float32Array | null;
+
+    /**
+     * The transform ID.
+     */
+    private _transformID: number;
+
+    /**
+     * The transform ID.
+     * @defaultValue `-1`
+     */
+    private _transformTrimmedID: number;
+
+    /**
+     * The tint applied to the sprite. This is a hex value. A value of 0xFFFFFF will remove any tint effect.
+     * @defaultValue `new PIXI.Color(0xFFFFFF)`
+     */
+    protected _tintColor: PIXI.Color;
+
+    /**
+     * The tint applied to the sprite. This is a RGB value. A value of 0xFFFFFF will remove any tint effect.
+     * @defaultValue `0xFFFFFF`
+     */
+    protected _tintRGB: number;
+
+    /**
+     * An instance of a texture uvs used for padded SpriteMesh.
+     * Instanced only when padding becomes non-zero.
+     * @defaultValue `null`
+     */
+    protected _textureUvs: PIXI.TextureUvs|null;
```

Comment:
I would've thought prettier would insert spaces here like ` | `. Is your editor set up correctly? I noticed some weird formatting on your last PR too.

Reply:
It *was*, not sure what happened.